bokeh/client/websocket.py

```diff
@@ -53,7 +57,7 @@ def __init__(self, socket):
         # messages atomically.
         self.write_lock = locks.Lock()
 
-    # Internal methods --------------------------------------------------------
```

Comment:
Would you rather call these "Internal methods" to be more pythonic or "Private methods"?

Reply:
I don't think we should try to organize/label method groups unsure classes with comments at all at this point, it's just a big task to get bogged down in, and also there has just not been any real thought about what groupings would be useful or make sense